recipes/libsecret/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Nasty hack for $PREFIX/bin/intltool-*
+ln -s "$PREFIX/bin/perl" "$PREFIX/bin/perl -w"
```

Comment:
Wow, yeah that's gross. I guess it's okay if necessary since it's only happening in the build env, though.

Reply:
Couldn't think of a shorter way, other than patching intltool itself.